test/mpi/maint/jenkins/xfail.conf

```diff
@@ -30,19 +30,6 @@
 
 ################################################################################
 
-# gather_big uses around 16GB of memory for the user buffers.  That
-# already puts the test outside of the memory capacity of our FreeBSD
-# nodes.  It is technically with the memory capacity of our remaining
-# nodes, but the MPICH algorithm for large Gathers allocates too much
-# temporary buffer space causing the remaining configurations to
-# intermittently fail too (see issue 3770 for more details).
-* * * * * sed -i "s+\(^gather_big .*\)+\1 xfail=issue3770+g" test/mpi/coll/testlist
-
-################################################################################
-# other failures on FreeBSD because of memory limits in our jenkins
-# infrastructure
-* * * * freebsd64 sed -i "s+\(^large_type_sendrec .*\)+\1 xfail=ticket0+g" test/mpi/datatype/testlist
```

Comment:
we'll have to make sure to test this with `--enable-large-tests`, which is currently set to "no" for review jobs.

Reply:
Hmm... it seems the it is enabled even for review tests.